auth/session.js

```diff
@@ -7,13 +7,17 @@ module.exports = () => ({
   config: {
     auth: false
   },
-  handler: (request, reply) => {
+  handler: (request, h) => {
     if (validate(request.payload.username, request.payload.password)) {
-      const session = {valid: true};
+      const session = { valid: true };
       const jwt = JWT.sign(session, config.JWT_SECRET);
-      return reply.redirect('/').state('token', jwt, {ttl: 5 * 3600 * 1000});
+      console.log('GOOD');
```

Comment:
please consider removing these `console.log('GOOD')` and `console.log('BAD')` or making them more specific so the person reading the logs can stack-trace/debug them. 😉